### ISSUE TITLE
[feature] allow multiple flags

### ISSUE THREAD

**Issue by moebiussurfing** [altschuler/imgui-knobs#6]
it seems that only one flag can be enabled, right?
multiple at same time would be super cool maybe.

```
static float value = 0;

ImGuiKnobFlags flags;
flags += ImGuiKnobFlags_ValueTooltip;
flags += ImGuiKnobFlags_NoTitle;
flags += ImGuiKnobFlags_NoInput;
flags += ImGuiKnobFlags_DragHorizontal;

if (ImGuiKnobs::Knob("Value", &value, -6.0f, 6.0f, 12 / 250.f, "%.1fdB", ImGuiKnobVariant_WiperDot, size, flags))
{
}
```

**Comment by altschuler** [comment]
```cpp
ImGuiKnobFlags flags = ImGuiKnobFlags_ValueTooltip | ImGuiKnobFlags_NoTitle | ImGuiKnobFlags_NoInput;
```
This uses the "bitwise or" (`|`) operator to create a so-called bitmask, sometimes called bitflags. This is why the flags in the sourcecode are all powers of two. Read https://stackoverflow.com/questions/18591924/how-to-use-bitmask for some more explanation.

**Comment by altschuler** [comment]
PS you can also use your style, just use `flags |= <flag>` instead, using `+=` will result in what would feel like random behavior.

**Comment by moebiussurfing** [comment]
I have already tested it with both ways ( | and + operators) but do not worked...
it seems that only ImGuiKnobFlags_ValueTooltip is acting.

```
static float value = 0;
ImGuiKnobFlags flags = ImGuiKnobFlags_ValueTooltip | ImGuiKnobFlags_NoTitle | ImGuiKnobFlags_NoInput;
if (ImGuiKnobs::Knob("Value", &value, -6.0f, 6.0f, 12 / 250.f, "%.1fdB", ImGuiKnobVariant_WiperDot, 100, flags))
{
}
```

**Comment by altschuler** [comment]
> I have already tested it with both ways ( | and + operators) but do not worked... it seems that only ImGuiKnobFlags_ValueTooltip is acting.
> 
> ```
> static float value = 0;
> ImGuiKnobFlags flags = ImGuiKnobFlags_ValueTooltip | ImGuiKnobFlags_NoTitle | ImGuiKnobFlags_NoInput;
> if (ImGuiKnobs::Knob("Value", &value, -6.0f, 6.0f, 12 / 250.f, "%.1fdB", ImGuiKnobVariant_WiperDot, 100, flags))
> {
> }
> ```

Sorry, I was too quick there, indeed it was a bug :) fixed in 1d31738d31f7f1840328de4a00756f58f668eb90.

Thanks for all your testing and help!
